src/aleph/db/accessors/aggregates.py

```diff
@@ -1,7 +1,7 @@
 import datetime as dt
 from typing import Optional, Iterable, Any, Dict, Tuple, Sequence
-
 from sqlalchemy import (
+    join,
```

Comment:
Unused import. Note that PyCharm can remove unused imports for you (Code > Optimize Imports)